resources/dicts/events/death/beach.json

```diff
@@ -174,16 +190,30 @@
         "weight": 20,
         "event_text": "m_c drowned after being pushed out to sea by o_c warriors.",
         "m_c": {
-            "age": [],
-            "status": [],
+            "age": [
+                "adolescent",
+                "young adult",
+                "adult",
+                "senior adult" 
+                ],
+            "status": [
+                "apprentice",
+                "warrior" ],
             "dies": true
         },
         "history": {
-            "reg_death": "m_c was pushed out to sea by enemy warriors."
+            "reg_death": "m_c was pushed out to sea by enemy warriors.",
+            "lead_death": "{VERB/m_c/were/was pushed out to sea by enemy warriors"
+        },
+        "other_clan": {
+          "current_rep": [
+            "hostile"
+          ],
+          "changed": -5
         }
     },
     {
-        "event_id": "bch_death_",
+        "event_id": "bch_death_heat_multiapp1",
```

Comment:
i know this event probably wasn't classic restricted before, but I think it probably should be.